Update Contributors.md

him9666 opened the pull request:
added the details for tree pattern

aritraroy24 commented:
@him9666 I'm closing the PR as it is not fulfilling the requirement...you can send a PR again after maintaining [all the steps](https://github.com/aritraroy24/HACKTOBERFEST2021_PATTERN#just-7-simple-steps-to-earn-your-t-shirt-for-hacktoberfest-2021-) and [eligibility criteria](https://github.com/aritraroy24/HACKTOBERFEST2021_PATTERN#-eligibility-criteria-) for a valid PR...

him9666 commented:
Hi Aritraroy, i have starred your repository, please remove the invalid label.  I think i have followed all other instructions you mentioned. added the photo, screenshot and corresponding code. Added my details in the contributors.md file. I am not sure what other conditions i missed . kindly let me know as soon as possible.
